UIforETW/Utility.h

```diff
@@ -108,24 +114,24 @@ class ElapsedTimer
 
 // High-precision timer class using QueryPerformanceCounter.
 // This may make ElapsedTimer unnecessary.
-class QPCElapsedTimer
+class QPCElapsedTimer final
 {
 public:
 	QPCElapsedTimer()
 	{
-		QueryPerformanceCounter(&start_);
+		ATLVERIFY(QueryPerformanceCounter(&start_));
 	}
 	double ElapsedSeconds() const
 	{
-		LARGE_INTEGER stop;
-		QueryPerformanceCounter(&stop);
-		LARGE_INTEGER frequency;
-		QueryPerformanceFrequency(&frequency);
+		LARGE_INTEGER stop = {0};

```

Comment:
Huh. I never noticed that. They should probably stick a couple of `_Always_` annotations on it.

For reference: [QueryPerformanceFrequency function](https://msdn.microsoft.com/en-us/library/windows/desktop/ms644905.aspx)